### PR TITLE
hack/run-locally.sh: don't open AWS ports for libvirt clusters

### PR DESCRIPTION
**Pull request by dcbw**
@danwinship @pecameron @JacobTanenbaum @rcarrillocruz @squeed

### CODE REVIEW

File: hack/run-locally.sh [hunk 1]
```diff
@@ -152,9 +154,20 @@ function create_cluster() {
...
+    else
+        echo "could not detect install platform from terraform.tfstate" >&2
+        exit 1
+    fi
+}
+
 # Fix up the AWS firewall so network plugin components can talk to each other
 function open_aws_ports() {
-    if [[ "${NETWORK_PLUGIN}" == "OVNKubernetes" ]]; then
+    if [[ "${NETWORK_PLUGIN}" == "OVNKubernetes" && "${PLATFORM}" == "aws" ]]; then
```
Comment (danwinship) [inline hack/run-locally.sh]: I was assuming that if multi-platform run-locally.sh support arrived before open-ovn-ports.sh went away that we'd add support for other platforms to open-ovn-ports.sh. Is there no firewalling in libvirt?

File: hack/run-locally.sh [hunk 2]
```diff
@@ -249,6 +263,7 @@ if [[ -z "$(which oc 2> /dev/null || exit 0)" ]]; then
     exit 1
 fi
 
+PLATFORM="aws"
```
Comment (danwinship) [inline hack/run-locally.sh]: hm?

File: hack/run-locally.sh [hunk 3]
```diff
@@ -152,9 +154,20 @@ function create_cluster() {
     trap "trap - TERM && kill -- -$$" INT TERM EXIT
 }
 
+function detect_platform() {
```
Comment (danwinship) [inline hack/run-locally.sh]: fwiw if you're only doing this when the cluster is up, you can do `oc get infrastructure cluster -o jsonpath='{.status.platform}'`

File: hack/run-locally.sh (hunk 1 above)
Comment (dcbw) [reply hack/run-locally.sh]: @danwinship there's whatever firewalling RHCOS does, which appears to be literally nothing. Like, all it has are nat+filter tables with default chain policies of ACCEPT. So I guess if that ever changes or it ever starts running firewalld we'd need to update this.

**Comment by dcbw** [comment]
Closing in favor of https://github.com/openshift/cluster-network-operator/pull/216

**Pull request closed without merging**